extensions/amp-base-carousel/0.1/test-e2e/test-goToSlide.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {getPrevArrow, getSlides} from './helpers';
+
+describes.endtoend(
+  'amp base carousel in lightbox go to slide',
+  {
+    fixture: 'amp-base-carousel/arrows-in-lightbox.amp.html',
+    environments: ['single'],
```

Comment:
Does this work on viewer or shadow demo environments?

Reply:
It dies, sadly.